core/src/main/java/tc/oc/pgm/listeners/ChatDispatcher.java

```diff
@@ -1,488 +1,64 @@
 import net.kyori.adventure.text.Component;
 import net.kyori.adventure.text.TextComponent;
 import net.kyori.adventure.text.format.NamedTextColor;
-import net.kyori.adventure.text.format.TextDecoration;
-import org.bukkit.Bukkit;
-import org.bukkit.command.CommandSender;
-import org.bukkit.entity.Player;
-import org.bukkit.event.EventHandler;
-import org.bukkit.event.EventPriority;
-import org.bukkit.event.Listener;
-import org.bukkit.event.player.AsyncPlayerChatEvent;
-import org.bukkit.event.player.PlayerQuitEvent;
-import org.incendo.cloud.annotation.specifier.Greedy;
-import org.incendo.cloud.annotations.Argument;
-import org.incendo.cloud.annotations.Command;
-import org.incendo.cloud.annotations.CommandDescription;
-import org.incendo.cloud.annotations.Permission;
-import org.jetbrains.annotations.NotNull;
-import org.jetbrains.annotations.Nullable;
 import tc.oc.pgm.api.PGM;
 import tc.oc.pgm.api.Permissions;
-import tc.oc.pgm.api.integration.Integration;
 import tc.oc.pgm.api.match.Match;
-import tc.oc.pgm.api.match.MatchManager;
-import tc.oc.pgm.api.party.Party;
 import tc.oc.pgm.api.player.MatchPlayer;
 import tc.oc.pgm.api.setting.SettingKey;
 import tc.oc.pgm.api.setting.SettingValue;
-import tc.oc.pgm.command.SettingCommand;
-import tc.oc.pgm.ffa.Tribute;
-import tc.oc.pgm.util.Audience;
-import tc.oc.pgm.util.Players;
-import tc.oc.pgm.util.bukkit.BukkitUtils;
-import tc.oc.pgm.util.bukkit.Sounds;
-import tc.oc.pgm.util.channels.Channel;
-import tc.oc.pgm.util.event.ChannelMessageEvent;
-import tc.oc.pgm.util.named.NameStyle;
-import tc.oc.pgm.util.text.TextException;
-import tc.oc.pgm.util.text.TextTranslations;
+import tc.oc.pgm.channels.AdminChannel;
+import tc.oc.pgm.channels.ChannelManager;
 
-public class ChatDispatcher implements Listener {
+@Deprecated()
+public class ChatDispatcher {
 
-  private static ChatDispatcher INSTANCE = new ChatDispatcher();
+  private static final ChatDispatcher INSTANCE = new ChatDispatcher();
 
   public static ChatDispatcher get() {
     return INSTANCE; // FIXME: no one should need to statically access ChatDispatcher, but community
     // does this a lot
   }
 
-  private final MatchManager manager;
-  private final Map<UUID, MessageSenderIdentity> lastMessagedBy;
-
   public static final TextComponent ADMIN_CHAT_PREFIX = text()
       .append(text("[", NamedTextColor.WHITE))
       .append(text("A", NamedTextColor.GOLD))
       .append(text("] ", NamedTextColor.WHITE))
       .build();
 
-  private static final String GLOBAL_SYMBOL = "!";
-  private static final String DM_SYMBOL = "@";
-  private static final String ADMIN_CHAT_SYMBOL = "$";
-
-  private static final String GLOBAL_FORMAT = "<%s>: %s";
-  private static final String PREFIX_FORMAT = "%s: %s";
-  private static final String AC_FORMAT =
-      TextTranslations.translateLegacy(ADMIN_CHAT_PREFIX) + PREFIX_FORMAT;
+  private static final Sound DM_SOUND = sound(key("random.orb"), Sound.Source.MASTER, 1f, 1.2f);
```

Comment:
Import `Sounds.DIRECT_MESSAGE` from the `Sounds` interface if it's needed; slipped through rebase?

Reply:
chat dispatcher is only kept for backwards-compat, it should simply not have any of this left